pkg/components/aws-ebs-csi-driver/component.go

```diff
@@ -39,6 +39,15 @@ enableVolumeScheduling: {{ .EnableVolumeScheduling }}
 enableVolumeResizing: {{ .EnableVolumeResizing }}
 # Enable volume snapshot.
 enableVolumeSnapshot: {{ .EnableVolumeSnapshot }}
+
+storageClasses:
+- name: ebs-sc
+  {{ if .EnableDefaultStorageClass }}
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
+  {{ end }}
+  volumeBindingMode: WaitForFirstConsumer
+  reclaimPolicy: Retain
```

Comment:
Why Retain as a default? This change has not been mentioned in commit message and it's a functional change.

Reply:
Added